src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java

```diff
@@ -46,12 +45,14 @@ public void parse() {
 
     private void download() {
         try {
-            File random = new File(Directories.getCachePath() + "/randoms.jar");
+            File random = new File(Directories.getCachePath() + File.separator + "randoms.jar");
```

Comment:
What if you want the nightly version, but you used to run the default version.
`randoms.jar` will still exist, though it won't download the nightly one, as it already exists.

Maybe first check if it's nightly, then append `nightly` to the file name, like `randoms-nightly.jar`.